ncm-systemd/src/main/perl/Systemd/Service/Unit.pm

```diff
@@ -442,8 +442,13 @@ sub configured_units
                 $self->error("Unitfile configuration failed, skipping the unit ", $self->unit_text($detail));
                 next;
             } elsif ($changed) {
-                $self->verbose("Going to issue conditional restart for $detail->{name} due to changed unitfile");
-                push(@tryrestart, $detail->{name});
+                if ($detail->{startstop}) {
+                    $self->verbose("Going to issue conditional restart for $detail->{name} due to changed unitfile");
+                    push(@tryrestart, $detail->{name});
+                } else {
+                    # Should we try to try-restart some dependency instead?
+                    $self->verbose("Changed unitfile for $detail->{name} but no conditional restart with false startstop");
```

Comment:
```suggestion
                    $self->verbose("Changed unitfile for $detail->{name} but service restart disabled via startstop configuration");
```

Reply:
ok, but would like to keep the conditional restart mentioned in it

Reply:
Have edited my suggestion.